### PR TITLE
SOW-24: visualisation example page

### PR DESCRIPTION
**Pull request by vferraro-scottlogic**
## Changes
- add new route for "visualisation example" page
- link "how to publish" page and "visualisation example" page

### CODE REVIEW

File: src/main.tsx [hunk 1]
```diff
@@ -86,6 +87,10 @@ const router = createBrowserRouter(
           path: PageRoute.Export,
           element: FormPageHoC(ExportPage, {}),
         },
+        {
+          path: PageRoute.VisualisationExample,
+          element: <VisualisationExamplePage />,
```
Comment (vferraro-scottlogic) [inline src/main.tsx:92]: I have created a new component for the page which is just a wrapper around the Visualisation component with paths to the assets files. I like having it as a component in pages/ for consistency but we could also keep it simple and use the Visualisation component here. i.e.:
{
  path: PageRoute.VisualisationExample,
  element: (
    <Visualisation
       developmentPlanFilepath={"assets/development-plan.csv"}
       timetableEventsFilepath={"assets/timetable.csv"}
     />
 ),
 },

Comment (Ro4052) [reply src/main.tsx:92]: No strong opinions on this :)

Comment (gcooperSL) [reply src/main.tsx:92]: could this be lazy loaded? just to keep in line with [this](https://github.com/digital-land/local-plans-timetable/pull/83) pr

Comment (vferraro-scottlogic) [reply src/main.tsx]: Done! Thanks George :)

**Pull request merged**